package.json

```
@@ -70,16 +77,29 @@
   "eslintConfig": {
     "extends": [
       "@wfcd"
-    ]
+    ],
+    "parserOptions": {
+      "requireConfigFile": false
+    },
+    "rules": {
+      "import/extensions": [
+        2,
+        "never",
+        {
+          "mjs": "always",
+          "json": "always"
+        }
+      ]
+    }
```

should be able to remove this w/ updated eslint-config
```suggestion
    }
```